lib/config.js

```diff
@@ -19,21 +24,10 @@ import React from 'react'
 export function configure(otpConfig) {
   return {
     /**
-     * Modify the customIcons object as needed
+     * (Optional) Define the base icon set to use/customize.
+     * Defaults to @opentripplanner/icons/ClassicModeIcon
      */
-    applyCustomIconLogic: customIcons => {
-      customIcons.customIconForLeg = (leg) => {
-        if (
-          leg.routeLongName &&
-          leg.routeLongName.startsWith('Portland Streetcar')
-        ) {
-          return 'STREETCAR'
-        } else if (leg.rentedBike) {
-          return 'BIKETOWN'
-        }
-        return null
-      }
-    },
+    baseModeIcon: ClassicModeIcon,
```

Comment:
@binh-dam-ibigroup, this version of `config.js` should represent the TriMet configuration (TriMet currently uses this repo with no config overrides). Given that, shouldn't this icon be Standard instead of Classic?

Reply:
You are pointing a gap in the solution I proposed!